src/ethereum/cancun/fork_types.py

```diff
@@ -116,7 +116,7 @@ class BlobTransaction:
     max_priority_fee_per_gas: Uint
     max_fee_per_gas: Uint
     gas: Uint
-    to: Union[Bytes0, Address]
+    to: Address
```

Comment:
Should we upstream a test for this?

Reply:
There is already a test for this in the latest `ethereum/tests`